clients/web/src/models/UserModel.js

```diff
@@ -1,6 +1,26 @@
 girder.models.UserModel = girder.Model.extend({
     resourceName: 'user',
 
+    // Sets this model to the currently logged in user; has no effect is no user
+    // is logged in.
+    current: function (success, error) {
+        girder.restRequest({
+            method: 'GET',
+            path: '/user/me'
+        }).then(_.bind(function (user) {
+            if (user) {
+                this.set(user);
+                if (success) {
+                    success(this);
+                }
+            } else {
+                if (error) {
+                    error(this);
+                }
```

Comment:
I think there are two possibilities:

1. Instead of success/error, we call either ``this.set()`` or ``this.clear()`` depending on whether ``user`` is ``null``.  Clients can react to the model's ``change`` event and then call ``isNew()`` etc. to see the result of the call to ``current()``.

2. We just dump this function altogether, since the same effect can be achieved by calling ``girder.fetchCurrentUser()`` followed by ``userModel.set()``/``userModel.clear()``.

The latter is obviously simpler, but my whole intent was to provide some abstraction to client writers, even if it's only a small amount - this instance would eliminate the need to check the result of ``fetchCurrentUser()`` by hand, for example.  I think that's useful but obviously I'll defer to your judgment.